caster/apps/atom.py

```diff
@@ -3,462 +3,408 @@
         "fold":
-            R(Key("ac-lbrace"), rdescript="Atom: Fold"),
+            ACK("ac-lbrace", "Atom: Fold"),
         "unfold":
-            R(Key("ac-rightbrace"), rdescript="Atom: Unfold"),
+            ACK("ac-rightbrace", "Atom: Unfold"),
         "unfold all":
-            R(Key("acs-rightbrace"), rdescript="Atom: Unfold All"),
+            ACK("c-k, c-0, acs-rightbrace", "Atom: Unfold All"),
         "fold [level] [<n2>]":
-            R(Key("c-%(n)s"), rdescript="Atom: Fold Level 1-9"),
+            ACK("c-k, c-%(n2)s", "Atom: Fold Level 1-9"),
         #Bookmarks Submenu
-        "view all":
-            R(Key("c-f2"), rdescript="Atom: Reflow Section"),
+        "view (all | [all] bookmarks)":
+            ACK("c-f2", "Atom: View All Bookmarks"),
         "bookmark | book":
-            R(Key("ca-f2"), rdescript="Atom: Toggle Bookmark"),
-        "next bookmark | next book":
-            R(Key("f2"), rdescript="Atom: Jump to Next Bookmark"),
-        "previous bookmark | previous book":
-            R(Key("s-f2"), rdescript="Atom: Jump to Previous Bookwork"),
+            ACK("ca-f2", "Atom: Toggle Bookmark"),
+        "next (bookmark | book)":
+            ACK("f2", "Atom: Jump to Next Bookmark"),
+        "previous (bookmark | book)":
+            ACK("s-f2", "Atom: Jump to Previous Bookwork"),
         #View Menu
         "reload file":
-            R(Key("ac-r"), rdescript="Atom: Reload File"),
+            ACK("ac-r", "Atom: Reload File"),
         "fullscreen":
-            R(Key("f11"), rdescript="Atom: Toggle Fullscreen"),
+            ACK("f11", "Atom: Toggle Fullscreen"),
         "toggle menubar":
-            R(palettized("Toggle Menu Bar"), rdescript="Atom: Toggle Menubar"),
+            ACP("Toggle Menu Bar", "Atom: Toggle Menubar"),
         "increase font [size] [<n>]":
-            R(Key("cs-equals"), rdescript="Atom: Increase Font Size")*Repeat(extra="n"),
+            ACK("cs-equals", "Atom: Increase Font Size")*Repeat(extra="n"),
         "decrease font [size] [<n>]":
-            R(Key("cs-minus"), rdescript="Atom: Decrease Font size")*Repeat(extra="n"),
+            ACK("cs-minus", "Atom: Decrease Font size")*Repeat(extra="n"),
         "reset font [size]":
-            R(Key("c-0"), rdescript="Atom: Reset Font Size"),
+            ACK("c-0", "Atom: Reset Font Size"),
         "toggle soft wrap":
-            R(palettized("Toggle Soft Wrap"), rdescript="Atom: Toggle Soft Wrap"),
-        ##"toggle command palette":                  R(Key(""), rdescript="Atom: Toggle Command Palette"),
+            ACP("Toggle Soft Wrap", "Atom: Toggle Soft Wrap"),
         "[toggle] treeview":
-            R(Key("c-backslash"), rdescript="Atom: Toggle Treeview"),
+            ACK("c-backslash", "Atom: Toggle Treeview"),
         #Panes Submenu
         "split above":
-            R(palettized("Pane: Split Up"), rdescript="Atom: Split Up"),
+            ACP("Pane: Split Up", "Atom: Split Up"),
```

Comment:
The "split" commands all have keyboard shortcuts - ctrl-k, up/down/left/right

Reply:
They both don't quite do the same thing. Those keyboard shortcuts split the pane,  then duplicate the currently open tab. The existing voice commands create blank panes. I'll gladly change it, though, because I think that behavior is more intuitive.

Reply:
Ah okay, entirely up to you then :)